beads_db/pom.xml

```diff
@@ -20,37 +20,7 @@
     </properties>
 
     <dependencies>
-        <dependency>
```

Comment:
would be good to put here dependency to flyway and out config file for it.
Otherwise we have inconsistence that sql scripts located here but jar and config file in other module